crypto/src/conversation.rs

```diff
@@ -59,6 +62,7 @@ impl MlsConversationConfiguration {
     }
 }
 
+/// This type will store the state of a group
```

Comment:
Be a bit more descriptive about what a conversation is, its link with a MLS group etc..